### PR TITLE
Create epidermalElectrophysiologyTechnique.jsonld

### PR DESCRIPTION
**Pull request by lzehl**
related to #352 

@UlrikeS91 the definition does not make me happy yet... I'm open for suggestions ;)

### CODE REVIEW

File: instances/technique/epidermalElectrophysiologyTechnique.jsonld [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/epidermalElectrophysiologyTechnique",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "The term 'epidermal electrophysiology technique' refers to any electrophysiology technique where electrodes are placed onto the skin of an organism to non-invasively (or minimally invasively) measure electrical functions of it's body.",
```
Comment (UlrikeS91) [inline instances/technique/epidermalElectrophysiologyTechnique.jsonld]: Suggestion: 
"'Epidermal electrophysiology technique' describes a subclass of electrophysiology techniques where one or several electrodes are placed on the outermost cell layer of an organism (epidermis) to measure electrical properties." 

(probably still room for improvements...)

File: instances/technique/epidermalElectrophysiologyTechnique.jsonld [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "epidermal electrophysiology technique",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "cutaneous electrophysiology",
+    "cutaneous electrophysiology technique",
```
Comment (UlrikeS91) [inline instances/technique/epidermalElectrophysiologyTechnique.jsonld]: Is this correct? These seem more like related terms and not exact synonyms. "cutaneous" is only "related to the skin", "epidermal" is more specific.

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/technique/epidermalElectrophysiologyTechnique.jsonld (hunk 2 above)
Comment (lzehl) [reply instances/technique/epidermalElectrophysiologyTechnique.jsonld]: not sure. This article seems to use it as a synonym: https://www.nature.com/articles/s41551-019-0365-8 What do you think?

File: instances/technique/epidermalElectrophysiologyTechnique.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/technique/epidermalElectrophysiologyTechnique.jsonld]: updated with small changes. thank you!

File: instances/technique/epidermalElectrophysiologyTechnique.jsonld (hunk 2 above)
Comment (UlrikeS91) [reply instances/technique/epidermalElectrophysiologyTechnique.jsonld]: Yeah, I also came across that one but they only use cutaneous once in the first sentence. I got the impression that they were speaking of cutaneous techniques in general. My interpretation was that epidermal techniques are a subclass of cutaneous techniques, but I could be wrong 😉

Comment (lzehl) [reply instances/technique/epidermalElectrophysiologyTechnique.jsonld]: don't know either. let's delete them (we can always add them again)

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**